cub/cub/device/device_copy.cuh

```diff
@@ -183,14 +183,16 @@ struct DeviceCopy
     // IDIV_CEIL(num_ranges, 64)
     using BlockOffsetT = uint32_t;
 
+    constexpr MemcpyOption memcpy_option = MemcpyOption::Copy;
+
     return detail::DispatchBatchMemcpy<
       InputIt,
       OutputIt,
       SizeIteratorT,
       RangeOffsetT,
       BlockOffsetT,
       detail::batch_memcpy::policy_hub<RangeOffsetT, BlockOffsetT>,
-      false>::Dispatch(d_temp_storage, temp_storage_bytes, input_it, output_it, sizes, num_ranges, stream);
+      memcpy_option>::Dispatch(d_temp_storage, temp_storage_bytes, input_it, output_it, sizes, num_ranges, stream);
```

Comment:
Nitpick: this could just be `MemcpyOption::Copy` rather than a separate variable

Reply:
Agreed 👍